Workade/Views&ViewModels/Gallery/GalleryDetailViewController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  GalleryDetailViewController.swift
+//  Workade
+//
+//  Created by 김예훈 on 2022/10/21.
+//
+
+import UIKit
+
+class GalleryDetailViewController: UIViewController {
+    
+    var image: UIImage?
+    
+    lazy var imageView: UIImageView = {
+        let imageView = UIImageView(image: image)
+        imageView.contentMode = .scaleAspectFit
+        imageView.backgroundColor = .clear
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return imageView
+    }()
+    
+    lazy var dismissButton: UIButton = {
+        let button = UIButton(type: UIButton.ButtonType.custom)
+        let imageConfig = UIImage.SymbolConfiguration(pointSize: 17, weight: .bold)
+        let xmarkImage = UIImage(systemName: "xmark", withConfiguration: imageConfig)
+        
+        let blur = UIVisualEffectView(effect: UIBlurEffect(style: .systemUltraThinMaterialLight))
+        blur.isUserInteractionEnabled = false
+        blur.clipsToBounds = true
+        blur.layer.cornerRadius = 22
+        
+        button.setImage(xmarkImage, for: .normal)
+        button.tintColor = .theme.background
+        button.layer.cornerRadius = 22
+        button.insertSubview(blur, at: 0)
+        button.addTarget(self, action: #selector(close), for: .touchUpInside)
+        
+        blur.translatesAutoresizingMaskIntoConstraints = false
+        button.translatesAutoresizingMaskIntoConstraints = false
+        
+        blur.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        
+        if let imageView = button.imageView {
+            button.bringSubviewToFront(imageView)
+        }
+        
+        return button
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupLayout()
+        self.view.backgroundColor = .clear
+    }
+    
+    func setupLayout() {
+
+        view.addSubview(imageView)
+        view.addSubview(dismissButton)
+        
+        let aspectR = (image?.size.width ?? 0) / (image?.size.height ?? 0)
```

Comment:
[P1]`image.size.height`의 값을 못 찾으면 0을 사용하는데 0으로 나누면 에러 발생!

Reply:
```suggestion
        var aspectR: CGFloat = 1
        if let image = image {
            aspectR = image.size.width / image.size.height
        }
```
이미지가 nil이면 1대1 비율을 가지도록 수정했습니다!! 좋은 지적 감사드립니다!